packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -584,6 +622,36 @@ class _RenderCupertinoSwitch extends RenderConstrainedBox {
     markNeedsPaint();
   }
 
+  bool get onOffSwitchLabels => _onOffSwitchLabels;
```

Comment:
nit: this could have been `(Color, Color)?`, so the widget doesn't repaint if the label is off and the colors change.